core/translate/expr.rs

```diff
@@ -23,13 +23,13 @@ fn emit_cond_jump(program: &mut ProgramBuilder, cond_meta: ConditionMetadata, re
         program.emit_insn(Insn::If {
             reg,
             target_pc: cond_meta.jump_target_when_true,
-            null_reg: reg,
+            jump_if_null: reg,
```

Comment:
Note: i didn't change this here to reduce the scope but we are misusing `null_reg` or `jump_if_null` or whatever in places that had it already. This is not supposed to be read from a register but it's a literal boolean flag parameter.

Opinions on whether I fix this in the same PR or leave a FIXME somewhere?

`jump_if_null: bool` was _added_ to Eq,Ne,Lt,Le,Gt and Ge.